homeassistant/components/zwave_js/light.py

```diff
@@ -357,6 +375,8 @@ def _calculate_color_values(self) -> None:
             self._supports_color = True
             # convert to HS
             self._hs_color = color_util.color_RGB_to_hs(red, green, blue)
+            # Light supports color, set color mode to hs
+            self._color_mode = COLOR_MODE_HS
```

Comment:
zwave is using raw RGB internally, we might want to flag `COLOR_MODE_RGB` instead.